h/views/client.py

```diff
@@ -85,6 +112,7 @@ def sidebar_app(request, extra=None):
     ga_client_tracking_id = settings.get('ga_client_tracking_id')
 
     ctx = _app_html_context(api_url=request.route_url('api.index'),
+                            client_boot_url=_resolve_client_boot_url(request),
```

Comment:
The client will have an `/embed.js` => `https://unpkg.com/hypothesis@0.X.Y` redirect cached locally but won't have an `https://unpkg.com/hypothesis` => `https://unpkg.com/hypothesis@0.X.Y` redirect cached locally because the service proxies the original redirect. So not resolving the URL would require the client to perform an extra request.

In an earlier version of this PR, I just had `/app.html` include `/embed.js` directly (which the client _will_ have cached). Come to think of it, I think I could just revert back to doing that.